R/trace.R

```diff
@@ -92,6 +92,13 @@ trace_back <- function(top = NULL, bottom = NULL) {
   trace <- new_trace(calls, parents)
   trace <- trace_trim_env(trace, frames, top)
 
+  if (is_installed("winch")) {
```

Comment:
This should probably be opt-in during development? I.e. `is_true(peek_option("rlang_trace_use_winch"))`.

Can you document what winch is doing? Is it making any assumptions about the trace internals? Note that these internals are not stable yet, we still have some work to do to match Gabor's traces.

Reply:
I'm adding and modifying rows in the data frame, relying on the internal structure. Would it be better to provide a drop-in replacement for `sys.calls()` and `sys.frames()` instead?